gensrc/thrift/MasterService.thrift

```diff
@@ -76,7 +76,7 @@ struct TDisk {
     1: required string root_path
     2: required Types.TSize disk_total_capacity
     // local used capacity
-    3: required Types.TSize data_used_capacity
+    3: optional Types.TSize data_used_capacity
```

Comment:
Consider that when upgrading, be applies the new code, but fe is still REQUIRED, which will cause an exception.

Reply:
Set it to required